registration.tmpl.cpp

```diff
@@ -43,6 +43,9 @@ std::shared_ptr<Pothos::Block> factory__${factory.name}(${factory.exported_facto
     % endif
     % for method in factory.block_methods:
     __pothos_block->registerCallable("${method.name}", Pothos::Callable(&${factory.namespace}::${factory.className}::${method.name}).bind(__orig_block_ref, 0));
+    % if not method.parameters and method.name not in ["start", "stop"]:
+    __pothos_block->registerProbe("${method.name}");
```

Comment:
Register probe also takes names for the signal and slot. Since gnuradio is very snake case and not camel, it may make sense to give it formatted signal and slot names that use underscores and lower case.

Reply:
Updated and rebased.

Reply:
Well I kind of meant the other way. All of the gr calls are snake case, so the generated signal and slot in the probe should also be snake case to match when they show up in the list of signals and slots.

Reply:
Ah, oops. I'll do that tonight.